src/stream-ciphers.rst

```diff
@@ -255,83 +255,81 @@ Conclusion
+problem, it is our ECB construction. Clearly, something better is needed.
 
 Block cipher modes of operation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-One of the more common ways of producing a :term:`stream cipher` is to use a
+A common way of producing a :term:`stream cipher` is by using a
 block cipher in a particular configuration. The compound system behaves
-like a :term:`stream cipher`. These configurations are commonly called
-:term:`mode of operation`\s. They aren't specific to a particular block cipher.
+like a :term:`stream cipher`. These configurations are known as
+:term:`mode of operation`\s. They are not specific to a particular block cipher.
 
-:term:`ECB mode`, which we've just seen, is the simplest such :term:`mode of operation`.
+:term:`ECB mode`, which we have just seen, is the simplest such :term:`mode of operation`.
 The letters ``ECB`` stand for electronic code book [#]_. For reasons
-we've already gone into, :term:`ECB mode` is very ineffective. Fortunately,
-there are plenty of other choices.
+we already discussed, :term:`ECB mode` is very ineffective. Fortunately,
+there plenty of other choices exist.
 
 .. [#]
-   Traditionally, modes of operation seem to be referred to by a
+   Traditionally, modes of operation are represented as a
    three-letter acronym.
 
 CBC mode
 ~~~~~~~~
 
-:term:`CBC mode`, which stands for cipher block chaining, is a very common
-:term:`mode of operation` where plaintext blocks are XORed with the previous
-ciphertext block before being encrypted by the block cipher.
-
-Of course, this leaves us with a problem for the first plaintext block:
-there is no previous ciphertext block to XOR it with. Instead, we pick
-an IV: a random number that takes the place of the “first” ciphertext in
-this construction. :term:`initialization vector`\s also appear in many other
-algorithms. An :term:`initialization vector` should be unpredictable; ideally,
-they will be cryptographically random. They do not have to be secret:
-IVs are typically just added to ciphertext messages in plaintext. It may
-sound contradictory that something has to be unpredictable, but doesn't
-have to be secret; it's important to remember that an attacker must not
-be able to predict *ahead of time* what a given IV will be. We will
+:term:`CBC mode`, cipher block chaining, is a common
+:term:`mode of operation`. This operation involves XORing plaintext blocks with the previous
+ciphertext block before it is encrypted by the block cipher.
+
+Of course, this process presents a problem for the first plaintext block:
+there is no previous ciphertext block to XOR it with. Instead, we select
+an IV: a random number in place of the “first” ciphertext.
+:term:`initialization vector`\s also appears in many
+algorithms. An :term:`initialization vector` should be unpredictable, ideally,
+cryptographically random. IVs do not have to be kept secret:
+they are simply added to ciphertext messages in plaintext. It may
```

Comment:
@lvh ,

"simply" is ok but I think I would rather have the original "typically just"

Reply:
yep agreed

Reply:
Kept the original